server/venueless/importers/conftool.py

```diff
@@ -418,7 +418,7 @@ def create_posters_from_conftool(
                 continue
 
             nonce += 1
-            download_url, preview_url = mirror_conftool_file(
+            downloadUrl, preview_url = mirror_conftool_file(
```

Comment:
In Python code, please use "snake_case", like `download_url`, not `downloadUrl`.

Reply:
right! will change it back into snake_case. thanks!